src/posts/lucky_1.0.0.cr

```diff
@@ -0,0 +1,135 @@
+
+    ### Docker setup
+
+    New Lucky apps ship with a default Docker setup allowing you to boot your app
+    for local development in Docker. This setup has been in a few versions, but until
+    this release, it's required some manual configurations to work.
+
+    We spent some time on this setup. Now when you generate a new Lucky app, the generated
+    dockerfile will be configured for your setup. If you generate an API only app, the dockerfile
+    no longer needs to install NodeJS.
+
+    After generating your new Lucky application, just run `docker compose up`, and Docker will
+    take care of the rest for you!
+
+    ### CockroachDB
+
+    If you haven't heard of [CockroachDB](https://www.cockroachlabs.com/product/), yet, now
+    is a great time as any! This amazing database uses a PostgreSQL interface allowing you
+    to connect as if you're just connecting to Postgres. Since it works just like Postgres,
+    this means we can support it natively with Avram.
+
+    This release comes with several fixes for CockroachDB as a few different community members
+    are actively using it. We will continue to make things smoother over time, but give it a shot
+    and let us know how it goes!
+
+    ## Roadmap
+
+    Roadmaps can be difficult to layout, especially with OSS. Things may change at anytime; however,
+    here's a small list of a few things that feel sound when it comes to what to expect next with Lucky.
+
+    ### Native Windows support
+
+    We have slowly started ensuring that the entire Lucky ecosystem is available
+    to work on Windows natively. You can follow along with in [Issue #1746](https://github.com/luckyframework/lucky/issues/1746).
+    If you're currently a developer using Windows, any help you can provide will be greatly beneficial,
+    to all of us.
+
+    ### Supporting other ORMs
+
+    For better support with other ORMs that allow you to choose alternate DBs, we'd love to
+    have a better interface for shard extensions with ORMs. Supporting all of the form helper
+    methods, file uploads, and features Avram gives you within Lucky, but for other ORM shards.
+
+    ### Better complex queries
+
+    Building more complex queries with Avram allowing you to use better casting, CTEs, dynamic
+    functions and more. This would become a happy medium place between using Query objects and
+    dropping down to raw SQL.
+
+    ### Asset Management
+
+    We would like to put some time into keeping our typesafe system for assets, but also
+    make swapping out build tools a lot easier. Lucky will always ship with some default,
+    but it shouldn't take you an entire day to swap it for something else.
+
+    ### Updated CLI UI/UX
+
+    One of the most requested features for the CLI is the ability to generate a default
+    application without a DB. This makes sense as the website itself is a Lucky app
+    without a DB. Doing this will also be a step towards supporting alternate ORMs.
```

Comment:
```suggestion
    application without a DB. This makes sense as our main Lucky website (the one you're reading now) is a Lucky app
    without a DB. Doing this will also be a step towards supporting alternate ORMs.
```